index.php

```diff
@@ -2,28 +2,32 @@
     $page['title'] = 'Purchase elementary OS';
 
     $page['script-plugins'] = array(
+        'https://cdn.jsdelivr.net/bluebird/3.4.1/bluebird.min.js',
         'https://cdn.jsdelivr.net/g/jquery.leanmodal2@2.5',
```

Comment:
`https://cdn.jsdelivr.net/bluebird/3.4.1/bluebird.min.js` and `https://cdn.jsdelivr.net/g/jquery.leanmodal2@2.5` can be combined into `https://cdn.jsdelivr.net/g/jquery.leanmodal2@2.5,bluebird@3.4.1`